.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/configuration-reference
+version: 2.1
+
+orbs:
+  # Check https://circleci.com/developer/orbs/orb/circleci/go for more details
+  go: circleci/go@1.7.2
+
+commands:
+  nuodb_insight_command:
+    steps:
+      - checkout
+      - go/install:
+          version: "1.20"
+      - run:
+          name: Install Dependencies
+          command: |
+            BASE_COMMIT=$(git merge-base HEAD master)
+            MODIFIED_FILES=$(git diff --name-only "${BASE_COMMIT}")
+            if ! echo "${MODIFIED_FILES}" | grep -qvE '(\.md$)'; then
+              echo "Only docs were updated, stopping build process."
+              exit
```

Comment:
I don't think this actually aborts the job. It just causes the current command to exit, which causes setup to be skipped. If you want to abort the job, invoke the following:
```sh
circleci-agent step halt
```